content/docs/enterprise/reference/configure.md

```diff
@@ -209,6 +209,78 @@ A [Namespace][namespace-concept] is a collection of users, groups, routes, and p
 When using an IdP without directory sync or when working with non-domain users, they will not show up in the look-ahead search. See [Non-Domain Users](/docs/enterprise/concepts#non-domain-users) for more information.
 :::
 
+## External Data
+
+This section lets [administrators](/docs/enterprise/concepts#admin) add and manage external data sources. Information from external data sources can be used to extend [policies](/docs/enterprise/reference/manage#policies-1).
+
+### Add or Edit External Data Source
+
+#### URL
+
+The path to the external data. The supported formats are:
+
+- JSON file containing an array of objects. each object **must** contain an `id` field.
+
+  ```json title="example JSON"
+  [
+    { "id": "id4@example.com", "user.id": "user4" },
+    { "id": "id5@example.com", "user.id": "user5" },
+    { "id": "id6@example.com", "user.id": "user6" }
+  ]
+  ```
+
+- CSV file, where the first row indicates the field names and subsequent rows are records. One of the fields **must** be an `id`.
+
+  ```json title="example CSV"
+  id,user.id
+  id1@example.com,user1
+  id2@example.com,user2
+  id3@example.com,user3
+  ```
+
+- A tar or zip file containing files of one of the formats above. The file path within the tar file specifies the record type, if not defined in the configuration. For example, in an arhive containing the following structure:
```

Comment:
```suggestion
- A tar or zip file containing files of one of the formats above. The file path within the tar file specifies the record type if not defined in the configuration. For example, in an archive containing the following structure:
```

Spellcheck. No strong feelings about the comma, personal choice.

Reply:
Good call on the typo. I'mma opt to keep the comma.